maintenance_evolution.Rmd

```diff
@@ -196,3 +196,19 @@ You don't have to change the tests of deprecated functions until they are made d
 
 Once a function is made defunct, its tests are simply removed.
 
+## Archiving packages {#archivalguidance}
+
+Software generally has a finite lifespan, and packages may eventually need to be archived. Archived packages are moved to a dedicated GitHub organization, [ropensci-archive](https://github.com/ropensci-archive). Prior to archiving, the contents of the README file should be moved to an alternative location (such as "README-OLD.md"), and replaced with minimal contents including the following:
+
+- Title with name of package
+- A repostatus.org badge of "unsupported" for formerly released packages, or "abandoned" for former concept or WIP packages.
+- One further sentence stating something like, "This package has been archived. The former README is now in README-old", with a link to the old README contents.
+
+An example of a minimal README in an archived package is in [ropensci-archive/monkeylearn](https://github.com/ropensci-archive/monkeylearn/blob/master/README.md). Once the README has been copied elsewhere and reduced to minimal form, the following steps should be followed:
+
+- Close issues with a sentence explaining the situation and linking to this guidance.
+- Disable issues on GitHub (under repo settings).
```

Comment:
I think this is useless if the repo is archived. this was a leftover from when I thought we wouldn't archive repo